server/matchmaker/matchmaker_queue.py

```diff
@@ -102,6 +102,8 @@ async def queue_pop_timer(self) -> None:
 
             number_of_matches = len(self._matches)
             metrics.matches.labels(self.name).set(number_of_matches)
+            number_of_items_in_queue = len(self.queue)
```

Comment:
+1 for descriptive names! Maybe replace `items` by `searches`?

Reply:
Done